pallets/transfer-allowlist/src/mock.rs

```diff
@@ -37,31 +40,9 @@ frame_support::construct_runtime!(
 #[derive_impl(frame_system::config_preludes::TestDefaultConfig as frame_system::DefaultConfig)]
 impl frame_system::Config for Runtime {
 	type AccountData = pallet_balances::AccountData<Balance>;
+	type AccountId = AccountId32;
 	type Block = frame_system::mocking::MockBlock<Runtime>;
-}
-
-#[derive(
-	Clone,
-	Debug,
-	PartialOrd,
-	Ord,
-	Encode,
-	Decode,
-	Eq,
-	PartialEq,
-	MaxEncodedLen,
-	TypeInfo,
-	Deserialize,
-	Serialize,
-)]
-pub enum Location {
```

Comment:
Personally, I liked this in order to not import centrifuge runtime types in pallets. Did you find some issues with the current `Location` used here?